### DISCUSSION TITLE
LIOJ1020判斷質數

### DISCUSSION THREAD

**Discussion by chadomdase** [Lidemy/forum#305] [程式導師實驗計畫] [answered]
![image](https://github.com/Lidemy/forum/assets/71286468/45ed8e06-c0a6-4936-8a61-c9014c0fe86e)
 
 
 
大家好，這段程式碼我是用Python寫的，雖然有給ChatGPT檢查過沒問題，但還是一直得到Wrong Answer，希望有高手指點一下，謝謝!

**Comment by cwc329** [comment]
我不太懂 python，不過 print() 裡面放 list 和 string 顯示的結果應該不一樣吧？

**Reply by chadomdase** [reply]
不好意思，您的意思是指輸出結果可能不會像題目那邊示範的output一樣會每行只顯示一個結果嗎

**Comment by allenliaooooooo** [comment, marked as answer]
第一行的數字代表有幾個數字
你是不是把第一行也判斷質數了？

**Reply by chadomdase** [reply]
!!拍謝，沒看清楚題目，改完馬上通過了哈哈，謝謝您!
